ice40/utils/fasm_icebox/asc2fasm.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import sys
+from fasm_icebox_utils import ascToFasm
```

Comment:
PEP8 violation?  ascToFasm should be asc_to_fasm?

Reply:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Done